src/functions_linearalgebra.jl

```diff
@@ -118,3 +118,38 @@ function Base.getproperty(
         return inner
     end
 end
+
+function LinearAlgebra.:\(
+    fact::NamedFactorization{L, T, F},
+    nda::NamedDimsArray{W},
+) where {L, T, F<:Factorization{T}, W}
+    n1, n2 = L
+    n1 != only(W) && throw(ArgumentError("Dimension mismatch with dimensions $L and $W"))
+    return NamedDimsArray{(n2,)}(LinearAlgebra.:\(parent(fact), parent(nda)))
+end
+
+function LinearAlgebra.:\(
+    fact::NamedFactorization{L, T, F},
+    nda::AbstractVector,
+) where {L, T, F<:Factorization{T}}
+    n1, n2 = L
+    return NamedDimsArray{(n2,)}(LinearAlgebra.:\(parent(fact), nda))
+end
+
+# Specialised routines for \ often do in-place ops that result in the nameddim populated from B
+# Leading to an incorrect named-dim
+for S in (UpperTriangular, LowerTriangular)
+    @eval begin
+        function LinearAlgebra.:\(A::$S{T, <:NamedDimsArray{L}}, B::NamedDimsArray) where {L, T}
+            n1, n2 = L
+            NamedDimsArray{(n2,)}(LinearAlgebra.:\($S(parent(parent(A))), parent(B)))
```

Comment:
{Upper/Lower}Triangular are wrappers and so parent gets called twice...is there a better way to do this?

Reply:
I went for setting up unname for a few types. 

Setting up recursively is probably it's own MR? AFAICT that's what you looked at [here](https://github.com/invenia/NamedDims.jl/pull/20) and were not sure about doing it.